satellite/console/consoleweb/server.go

```diff
@@ -117,7 +117,7 @@ func NewServer(logger *zap.Logger, config Config, service *console.Service, mail
 
 // appHandler is web app http handler function
 func (s *Server) appHandler(w http.ResponseWriter, req *http.Request) {
-	http.ServeFile(w, req, filepath.Join(s.config.StaticDir, "dist", "public", "index.html"))
+	http.ServeFile(w, req, filepath.Join(s.config.StaticDir, "dist", "index.html"))
```

Comment:
Can't we make such that it is always relative to the dir... i.e. `filepath.Join(s.config.StaticDir, "index.html")`?